src/components/CreateBlogPost_HTML.tsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import '../CreateBlogPost.css' //allows us to use the css from our other file
+import '../CreateBlogPost_HTML.css' //allows us to use the css from our other file
```

Comment:
shouldn't this be `./`? where is the css file?

Reply:
in src directory. This was Week 2 Code

Reply:
Can we make a css directory? or is that not good practice?